docs/src/main/sphinx/connector/delta-lake.rst

```diff
@@ -248,6 +248,9 @@ configure processing of Parquet files.
     * - ``parquet_writer_page_size``
       - The maximum page size created by the Parquet writer.
       - ``1MB``
+    * - ``parquet_writer_batch_size``
+      - Maximum number of rows processed by the parquet writer in a batch.
+      - 10,000
```

Comment:
I think this needs to be wrapped in quotes like in above properties

Reply:
The numerical constants are not wrapped.

https://github.com/trinodb/trino/blame/master/docs/src/main/sphinx/connector/hive.rst#L329

Reply:
hmm... there are also examples of numbers wrapped in quotes on fault-tolerant-execution.rst 
@mosabua what's the right way here ?

Reply:
Unfortunately its inconsistent at this stage. However.. whatever you do .. the value has to be a valid number .. and I dont think `10.000` is .. instead it would have to be `10000` .. this is important since users will try to modify the example value to e.g. `25,000` and then that would fail potentially.